consensus/clique/clique.go

```diff
@@ -725,7 +735,7 @@ func CliqueRLP(header *types.Header) []byte {
 }
 
 func encodeSigHeader(w io.Writer, header *types.Header) {
-	err := rlp.Encode(w, []interface{}{
+	enc := []interface{}{
```

Comment:
@fjl can we use `optional` here too ?

Reply:
It cannot be used here because this is not a struct.